### PR TITLE
Display IME unspecified string

### PR DESCRIPTION
**Pull request by lispcoc**
IME unspecified string is invisible now because SDL_TEXTEDITING event is not handled.
So handle SDL_TEXTEDITING event and make IME unspecified string visible.

### CODE REVIEW

File: src/sdltiles.cpp [hunk 1]
```diff
@@ -1306,6 +1319,7 @@ void CheckMessages()
                 quit = true;
                 break;
         }
+        if (text_reflesh) break;
```
Comment (Coolthulhu) [inline src/sdltiles.cpp]: Styling:
```
if( x ) {
    // Stuff
}
```

Braces shouldn't be skipped, parentheses should be padded on the inside, no space between `if` and parentheses.

File: src/string_input_popup.cpp [hunk 2]
```diff
@@ -347,6 +355,20 @@ const std::string &string_input_popup::query_string( const bool loop, const bool
             ret.insert( _position, t );
             _position += t.length();
             redraw = true;
+            edit.erase(0);
+            ctxt->set_edittext(edit.c_str());
+        }
+        else if (ev.edit_refresh) {
```
Comment (Coolthulhu) [inline src/string_input_popup.cpp]: Styling: `} else if( x ) {` - closing braces, `else` and `if` should be on one line.

File: src/sdltiles.cpp (hunk 1 above)
Comment (Coolthulhu) [reply src/sdltiles.cpp]: Also, looks like a typo

File: src/sdltiles.cpp [hunk 3]
```diff
@@ -1306,6 +1319,9 @@ void CheckMessages()
                 quit = true;
                 break;
         }
+        if(text_refresh) {
```
Comment (Night-Pryanik) [inline src/sdltiles.cpp]: Spaces inside the brackets.

**Pull request merged**